web/src/lib/components/admin-page/jobs/jobs-panel.svelte

```diff
@@ -125,7 +126,11 @@
 {/if}
 
 <div class="flex flex-col gap-7">
+<<<<<<< HEAD
 	{#each jobDetailsArray as [jobName, { title, subtitle, allText, missingText, allowForceCommand, component, handleCommand: handleCommandOverride }]}
+=======
+	{#each jobDetailsArray as [jobName, { title, subtitle, allowForceCommand, icon, component, handleCommand: handleCommandOverride }]}
+>>>>>>> upstream/main
```

Comment:
Missed merge conflict here btw.

Reply:
Ah, I did resolve this but it didn't get pushed up. Thanks/.